monitor/extensions/referencedata-wizards/src/main/java/org/datacleaner/monitor/server/wizard/shared/file/FileWizardSession.java

```diff
@@ -52,14 +52,6 @@ public Integer getPageCount() {
         return 1;
     }
 
-    @Override
-    protected Element getUpdatedReferenceDataSubSection(final DocumentBuilder documentBuilder) {
-        copyUploadedFileToReferenceDataDirectory();
-        return addElementToConfiguration();
-    }
-
-    protected abstract Element addElementToConfiguration();
-
```

Comment:
Hmmm... That removal might actually have been a mistake. I'll have to look into that.